src/components/SingleMovie.js

```diff
@@ -37,24 +37,30 @@ class SingleMovie extends React.Component {
       <div className="SingleMovie" style={{
           backgroundImage:`linear-gradient(to bottom, transparent 0%, black 60%),
           url(${backdrop_path})`,
-          backgroundPosition: 'center',
-          backgroundSize: 'cover',
-          backgroundRepeat: 'no-repeat',
-          width:'100vw',
-          height:'100vh'
           }}>
           <h3 className="single-movie-title"> 
             {title}
             <p className="single-movie-tagline"> {tagline}</p>
           </h3>
-          <div className="single-movie-info-container">
-            <p className="single-movie-overview">Overview:<br></br>{overview}</p>
-            <p>Average User Rating: {average_rating ? rating : average_rating}</p>
-            <div className="single-movie-info">
-              <p>Release Date: {dayjs(release_date).format('MMMM D, YYYY')}</p>
-              {!!genres && <p>Genre: {genres.join(", ")}</p>} 
-              {!!revenue && <p>Revenue: ${revenue}</p>} 
-              {!!budget  && <p>Budget: ${budget}</p>}
+          <div className="poster-info-container">
+            <div className="single-movie-poster">
+                <img 
+                  src={poster_path}
```

Comment:
I'm so stoked to see the poster in there! This really looks like a full blown website now, thank you so much for adding alt text too!!